Description: Do not use six
Author: Thomas Goirand <zigo@debian.org>
Bug-Debian: https://bugs.debian.org/1060153
Forwarded: no
Last-Update: 2024-02-20

--- python-pbr-6.0.0.orig/doc/requirements.txt
+++ python-pbr-6.0.0/doc/requirements.txt
@@ -3,4 +3,3 @@ sphinx!=1.6.6,!=1.6.7,>=1.6.2;python_ver
 sphinxcontrib-apidoc>=0.2.0  # BSD
 openstackdocstheme>=1.18.1 # Apache-2.0
 reno>=2.5.0 # Apache-2.0
-six==1.12.0 # MIT
--- python-pbr-6.0.0.orig/pbr/sphinxext.py
+++ python-pbr-6.0.0/pbr/sphinxext.py
@@ -15,7 +15,7 @@
 
 import os.path
 
-from six.moves import configparser
+import configparser
 from sphinx.util import logging
 
 import pbr.version
--- python-pbr-6.0.0.orig/pbr/tests/test_packaging.py
+++ python-pbr-6.0.0/pbr/tests/test_packaging.py
@@ -52,7 +52,6 @@ try:
 except ImportError:
     import mock
 import pkg_resources
-import six
 import testscenarios
 import testtools
 from testtools import matchers
@@ -218,18 +217,18 @@ class CreatePackages(fixtures.Fixture):
     """
 
     defaults = {
-        'setup.py': textwrap.dedent(six.u("""\
+        'setup.py': textwrap.dedent("""\
             #!/usr/bin/env python
             import setuptools
             setuptools.setup(
                 setup_requires=['pbr'],
                 pbr=True,
             )
-        """)),
-        'setup.cfg': textwrap.dedent(six.u("""\
+        """),
+        'setup.cfg': textwrap.dedent("""\
             [metadata]
             name = {pkg_name}
-        """))
+        """)
     }
 
     def __init__(self, packages):
--- python-pbr-6.0.0.orig/pbr/tests/test_util.py
+++ python-pbr-6.0.0/pbr/tests/test_util.py
@@ -17,8 +17,7 @@ import io
 import tempfile
 import textwrap
 
-import six
-from six.moves import configparser
+import configparser
 import sys
 
 from pbr.tests import base
@@ -27,7 +26,7 @@ from pbr import util
 
 def config_from_ini(ini):
     config = {}
-    ini = textwrap.dedent(six.u(ini))
+    ini = textwrap.dedent(str(ini))
     if sys.version_info >= (3, 2):
         parser = configparser.ConfigParser()
         parser.read_file(io.StringIO(ini))
--- python-pbr-6.0.0.orig/test-requirements.txt
+++ python-pbr-6.0.0/test-requirements.txt
@@ -6,7 +6,6 @@ wheel>=0.32.0 # MIT
 fixtures>=3.0.0 # Apache-2.0/BSD
 hacking>=1.1.0,<4.0.0;python_version>='3.6' # Apache-2.0
 mock>=2.0.0,<4.0.0;python_version=='2.7' # BSD
-six>=1.12.0 # MIT
 stestr>=2.1.0,<3.0;python_version=='2.7' # Apache-2.0
 stestr>=2.1.0;python_version>='3.0' # Apache-2.0
 testresources>=2.0.0 # Apache-2.0/BSD
